sklearn/feature_selection/_from_model.py

```diff
@@ -99,8 +100,15 @@ class SelectFromModel(MetaEstimatorMixin, SelectorMixin, BaseEstimator):
         ``threshold`` in the case where the ``coef_`` attribute of the
         estimator is of dimension 2.
 
-    max_features : int, default=None
-        The maximum number of features to select.
+    max_features : int, callable, default=None
+        The maximum number of features to select. ``max_features`` may be a
```

Comment:
I think a good example of describing multiple options is `importance_getter` below it. We can break the options apart into two sections.